packages/relay/src/lib/clients/mirrorNodeClient.ts

```diff
@@ -238,7 +239,14 @@ export class MirrorNodeClient {
         }
 
         this.logger.error(new Error(error.message), `${requestIdPrefix} [${method}] ${path} ${effectiveStatusCode} status`);
-        throw new MirrorNodeClientError(error.message, effectiveStatusCode);
+
+        const mirrorError = new MirrorNodeClientError(error, effectiveStatusCode);
+
+        if (mirrorError.isContractReverted() && !mirrorError.isNotSupported()) {
+            throw predefined.CONTRACT_REVERT(mirrorError.errorMessage);
```

Comment:
nit: comment here highlighting we want to distinguish unsupported mirror node evm functionality.

Reply:
Added a comment.